gptme/tools/shell.py

```diff
@@ -289,13 +289,15 @@ def execute_shell_impl(
 
     stdout = _shorten_stdout(stdout.strip(), pre_tokens=2000, post_tokens=8000)
     stderr = _shorten_stdout(stderr.strip(), pre_tokens=2000, post_tokens=2000)
-
     msg = (
-        _format_block_smart(
-            f"Ran {'allowlisted ' if allowlisted else ''}command", cmd, lang="bash"
-        )
-        + "\n\n"
+    _format_block_smart(
+        f"{'Failed to run' if returncode else 'Ran'} "
```

Comment:
I think "failed to run" might be bad prompting. We should be careful to not infer too much from a non-zero exit code I think.

Reply:
Removed the shell.py changes.